webview_panels/src/lib/altimate/altimate-components.d.ts

```
@@ -66,9 +67,16 @@ export declare interface CoachAiResponse {
     personalizationScope: string;
 }
 
-export declare const CoachForm: ({ taskLabel, context, onClose, extra }: Props_10) => JSX_2.Element;
+export declare const CoachForm: (props: CoachFormProps) => JSX_2.Element;
 
-export declare const CoachFormButton: ({}: Props_11) => JSX_2.Element;
+export declare const CoachFormButton: ({}: Props_10) => JSX_2.Element;
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)